src/ui/search-tab.h

```diff
@@ -0,0 +1,66 @@
+#ifndef SEAFILE_CLIENT_UI_SEARCH_TAB_H
+#define SEAFILE_CLIENT_UI_SEARCH_TAB_H
+#include "tab-view.h"
+
+class QWidget;
+class QLabel;
+class QListView;
+class SearchResultListView;
+class SearchResultListModel;
+class SearchResultItemDelegate;
+class QListWidget;
+class QListWidgetItem;
+class QLineEdit;
+class QTimer;
+class QModelIndex;
+class FileSearchRequest;
+struct FileSearchResult;
+class ApiError;
+class SearchTab : public TabView {
+    Q_OBJECT
+public:
+    explicit SearchTab(QWidget *parent = 0);
+    ~SearchTab();
+
+public slots:
+    void refresh() {}
```

Comment:
`SearchTab` should also implement `refresh` so users can refresh the search results by clicking the refresh button.